app/main.py

```diff
@@ -1 +1,39 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry is True:
```

Comment:
We can't use `is` keyword with operation upon values (is_hungry: bool), because `is` addresses to the memory. Use not or `==` instead